animation_url/src/routes/[slug]/+page.svelte

```diff
@@ -86,6 +86,7 @@
     right: 0;
     max-height: 37%;
     max-width: 25%;
+    min-width: 25%;
```

Comment:
@datadanne the garage pilot display might need something like this that allows for non-square media. I believe it assumes images are square?

Reply:
Ah, I was going to ask you how you handle this in this app yesterday. The garage has a fixed square of 25% x 25% for the pilot and displays the image inside that with `object-fit: contain`. So if you use an NFT that isn't square there will just be dark bg color padding so that the pilot box is still a square

Reply:
nice, changed to do the same here.